ibis/backends/clickhouse/tests/snapshots/test_select/test_simple_joins/join_keys1-inner_join/out.sql

```diff
@@ -0,0 +1,5 @@
+SELECT
+  batting.*
```

Comment:
Shouldn't this be `t1`?

Reply:
It might be an old snapshot, lemme delete and try again.